main.php

```diff
@@ -117,6 +117,8 @@
 require_once('core/startup/ErrorControlChain.php');
 require_once('core/startup/ParameterConfirmationToken.php');
 
+ini_set('display_errors', 0);
```

Comment:
OK I stand corrected: moving the display_errors supression to HTTPOutputHandler works a treat.